web/packages/design/src/Icon/Icons/Detective.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Teleport
+ * Copyright (C) 2023  Gravitational, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2025  Gravitational, Inc.
```

Reply:
This is due to the icon generation script being out of date - https://github.com/gravitational/teleport/blob/master/web/packages/design/src/Icon/script/script.js#L116